config/initializers/okcomputer.rb

```diff
@@ -61,9 +62,7 @@ def profile
 OkComputer::Registry.register 'suri_url', OkComputer::HttpCheck.new(Settings.SURI.URL)
 
 # Stacks
-OkComputer::Registry.register 'stacks_doc_cache_storage_root', OkComputer::DirectoryCheck.new(Settings.STACKS.DOCUMENT_CACHE_STORAGE_ROOT)
```

Comment:
so are we saying that these aren't on -dev environment, so they're not worth checking?